app/sidecar/base.toml

```diff
@@ -78,10 +78,10 @@ task-slots = ["sys"]
 
 [tasks.net]
 name = "task-net"
-stacksize = 8000
+stacksize = 10000
```

Comment:
What happened here?

Reply:
Not sure!  `MAXDEPTH` goes from 6160 → 8392, so it's a noticeable chance.